internal/acceptance/model_serving_test.go

```diff
@@ -122,6 +122,15 @@ func TestAccModelServing(t *testing.T) {
 					}
 				}
 			}
+
```

Comment:
I was informed this test is broken, so I'm surprised it works now?

Reply:
I just modified code, code is still broken :-)